experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/plugins.txt

```diff
@@ -0,0 +1,20 @@
+configuration-as-code
+cloudbees-folder
+antisamy-markup-formatter
+build-timeout
+credentials-binding
+timestamper
+ws-cleanup
+ant
+ssh-slaves
+gradle
+workflow-aggregator
+github-branch-source
+pipeline-stage-view
+git
+matrix-auth
+pam-auth
+pipeline-github-lib
+ssh-agent
+ldap
```

Comment:
Please remove unused plugins.

Reply:
I've made the changes Please take a look

Reply:
Good job.